cdc_kafka/constants.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+
+from .tracked_tables import ChangeTableIndex
+
+CHANGE_ROW_BATCH_SIZE = 1000
+TABLE_POLL_INTERVAL = datetime.timedelta(seconds=3)
```

Comment:
Are these constants based on cdc/kafka or based on anecdotal experience?

Reply:
Mostly the latter. We just want _some_ hedge against spamming the DB with rapid-fire queries in cases where the CDC traffic itself isn't providing a "natural" throttle.

Reply:
In that case, should these maybe be configurable rather than cosntants?